src/components/tasks/card/index.tsx

```diff
@@ -611,32 +591,77 @@ const Card: FC<CardProps> = ({
                           </div>
                       )
                     : editedTaskDetails.assignee && (
-                          <p className={styles.cardStrongFont}>
-                              {editedTaskDetails.assignee}
+                          <p className={styles.contributorDiv}>
+                              <span className={styles.contributor}>
+                                  <Image
+                                      src={assigneeProfileImageURL}
+                                      alt={cardDetails.assignee || DUMMY_NAME}
+                                      width={25}
+                                      height={25}
+                                  />
+                              </span>
+                              <span className={styles.cardStrongFont}>
+                                  {editedTaskDetails.assignee}
+                              </span>
                           </p>
                       )}
                 {showAssignButton() && <AssigneeButton />}
             </div>
 
-            <div className={styles.cardItems}>
-                <div
-                    className={`${styles.taskTagLevelWrapper} ${
-                        isEditable && styles.editMode
-                    }`}
-                >
-                    <TaskLevelMap
+            {/* EDIT task status */}
+            {isEditable ? (
+                <TaskStatusEditMode
+                    task={editedTaskDetails}
+                    setEditedTaskDetails={setEditedTaskDetails}
+                    isDevMode={isDevMode}
+                />
+            ) : (
+                <div className={styles.statusContainer} style={{}}>
+                    <p
+                        data-testid="task-status"
+                        className={`${styles.statusText} ${
+                            styles[
+                                `statusText${handleStatusTextColor(
+                                    cardDetails.status
+                                )}`
+                            ]
+                        } `}
+                    >
+                        <p
+                            className={`${styles.statusTextIndicator} ${
+                                styles[
+                                    `statusTextIndicator${handleStatusTextColor(
```

Comment:
same as above

Reply:
handleStatusTextColor is returning the name, that is the diff.

Reply:
we can keep it in a variable outside jsx for better readability